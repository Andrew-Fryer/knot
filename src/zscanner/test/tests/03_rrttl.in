$ORIGIN .
$TTL    1

; OK
@   0           NS @    ; Minimal ttl
@   3600        NS @    ; Inner value
@   4294967295  NS @    ; Maximal ttl

; KO
@   -1          NS @    ; Negative ttl
@   4294967296  NS @    ; 32bit overflow
