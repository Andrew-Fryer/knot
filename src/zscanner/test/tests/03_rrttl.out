OWNER=00
CLASS=0001
RRTTL=00000000
RTYPE=0002
RDATA=00
------
OWNER=00
CLASS=0001
RRTTL=00000E10
RTYPE=0002
RDATA=00
------
OWNER=00
CLASS=0001
RRTTL=FFFFFFFF
RTYPE=0002
RDATA=00
------
ERROR=ZSCANNER_EUNSUPPORTED_TYPE
------
ERROR=ZSCANNER_ENUMBER32_OVERFLOW
------

